backend/spiritual_avatar_generation_engine.py

```
@@ -457,8 +458,21 @@ async def test_avatar_services(self) -> Dict[str, Any]:
         
         return results
 
-# Global instance
-avatar_engine = SwamjiAvatarGenerationEngine()
+def get_avatar_engine(
+    settings: AppSettings = Depends(get_app_settings),
```

Ruff (0.12.2)

Redefinition of unused `settings` from line 22

(F811)